src/ButterCMS/ButterCMS.php

```diff
@@ -126,6 +134,22 @@ public function fetchTags($params = [])
     }
 
     ///////////////
+    // Pages
+    ///////////////
+
+    public function fetchPage($type, $slug, array $params = [])
+    {
+        $rawPage = $this->request('pages/' . $type . '/' . $slug . '/', $params);
+        return $rawPage ? new PageResponse($rawPage) : false;
```

Comment:
What's the point of this ternary?

Reply:
Technically the `request()` method can return `false` if there is an error parsing the JSON for whatever reason. I imagine that would be very uncommon unless there was some larger issue going on with the API. I could update that to throw an exception instead as it does for other errors so there is only one form of error handling needed by whoever is using the wrapper.

Reply:
Yeah I think throwing an exception is the correct behavior.  This should also be consistent across all calls. Why would we not do the same thing for `fetchPages`?

Reply:
Updated in https://github.com/ButterCMS/buttercms-php/pull/4/commits/1ff6f2a410bfd9c1aec3c601e96ac0d9382e704a. Correct, all of them now behave this way.